rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -475,15 +477,36 @@ public void processPegoutsInBatch_before_rskip_271() throws IOException {
     }
 
     @Test
-    public void processPegoutsInBatch_after_rskip_271() throws IOException {
+    public void test_processPegoutsInBatch_after_rskip_271() throws IOException {
```

Comment:
I think these tests are good but I have one concern though...
I see that in this test we are able to create the pegout because we have 3 requests, but in the second one we fail because we stack 5 requests... There is no clear indication on why this happens, unless you read the setup logic where there is one UTXO with 4 bitcoins.
I would like to have some clear indications in each test to let the reviewer/reader understand why one scenario fails and the other succeeds.

Thanks!

Reply:
Resolved

Reply:
Resolved